cmd/kubeadm/app/phases/addons/dns/manifests.go

```diff
@@ -222,6 +259,9 @@ metadata:
   namespace: kube-system
   labels:
     k8s-app: kube-dns
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
```

Comment:
kubeadm doesn't need these two.

these are addonmanager semantics [1], see:
https://github.com/kubernetes/kubernetes/issues/72757

Reply:
I see, thank you for pointing this out!